src/App.js

```diff
@@ -1,19 +1,106 @@
 import React from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList';
+import { NamesList } from './components/NamesList';
+import { AddToDoButton } from './components/AddToDoButton/AddToDoButton';
 
+import todos from './api/todos';
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const copyTodos = [...todos].map(todo => ({
```

Comment:
```suggestion
const copyTodos = todos.map(todo => ({
```
map - не мутирующий метод. не нужно создавать копию перед map